### DISCUSSION TITLE
New diffing algorithm doesn't update properties like el.value or el.checked . It just updates the elements attribute but not the node object properties

### DISCUSSION THREAD

**Discussion by joydeep-bhowmik** [livewire/livewire#6064] [7. Bugs (archived)]
### Livewire version

V3 beta

### Which PHP version are you using?

PHP 8.2

**Comment by PhiloNL** [comment]
Hi there! Can you please provide an example and step by step instructions on how to reproduce the error or submit a failing test. Thanks!

**Reply by joydeep-bhowmik** [reply]
blade view
```blade
@foreach ($photos as $item)
            <div class="flex flex-col gap-2">
                <input type="number" id="ordering_{{$item->id}}" value="" wire:change='change_order({{$item->id}}, $el.value)'>
                <div class="flex">
                    <button type="button" wire:click="delete({{$item->id}})">
                        <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6">
                            <path stroke-linecap="round" stroke-linejoin="round" d="M6 18L18 6M6 6l12 12" />
                        </svg>
                    </button>
                    <label for="ordering_{{$item->id}}" class="ml-auto bg-black text-white font-semibold rounded-full w-6 h-6 flex justify-center items-center">
                        {{$item->ordering}}
                    </label>
                </div>
                <img src="{{ asset('/images/'.$item->image_url) }}" alt="" class="h-auto max-w-full rounded-lg">
            </div>
@endforeach
```
Whenever the ordering is updated, all DOM elements are updated as expected. However, there is an issue with the input value not being updated in the new Livewire version (v3 beta). This issue was not present in Livewire 2 because it utilized morphDom for diffing, which ensured that node objects' properties were updated correctly. It is likely that the current diffing algorithm in the new version is causing the problem with the input value not being updated as expected.

**Reply by joshhanley** [reply]
@joydeep-bhowmik can you provide a Livewire component class and view, that we can copy, with instructions that demonstrates the issue so we can investigate? Thanks!

**Reply by joydeep-bhowmik** [reply]
I think this is same as #6091 so I'm closing this
